### PR TITLE
Fix grammar issues

### PR DESCRIPTION
**Pull request by stanbar**
I've checked to the Verdict point. You should consider splitting it to two posts, one for evaluating different key manager apps, and second for describing your setup.

### CODE REVIEW

**Comment by gbaranski** [comment]
I think it should stay together, it is just an explanation why I've chosen `gopass` over other password managers

File: content/posts/password-managers.md [hunk 1]
```diff
@@ -1,53 +1,54 @@
 +++
-title = "My setup & thoughts about password managers"
+title = "On password managers"
```
Comment (gbaranski) [inline content/posts/password-managers.md:2]: What about this?
```suggestion
title = "Comparison of password managers and my setup"
```

Comment (stanbar) [reply content/posts/password-managers.md:2]: Yes, much better.

**Comment by gbaranski** [comment]
Thanks for this PR.

**Pull request merged**